src/benchmark_environments/testing/envs.py

```diff
@@ -0,0 +1,185 @@
+"""Helper methods for tests of custom Gym environments.
+
+This is used in our test suite in `tests/test_envs.py`. It is also used in sister
+projects such as `imitation`, and may be useful in other codebases.
+"""
+
+import re
+from typing import (
+    Any,
+    Callable,
+    Iterable,
+    Iterator,
+    Mapping,
+    Optional,
+    Sequence,
+    Tuple,
+)
+
+import gym
+import numpy as np
+
+Rollout = Sequence[Tuple[Any, Optional[float], bool, Mapping[str, Any]]]
```

Comment:
```suggestion
Rollout = Sequence[Tuple[Any, Optional[float], bool, Mapping[str, Any]]]
"""A sequence of 4-tuples (obs, rew, done, info) as returned by `get_rollout`."""
```

Reply:
~~I don't remember `rew` ever being optional here. But info is definitely Optional.~~

~~I'd prefer `info`'s type to be `Optional[dict]` over the `Mapping` thing. (Arguably easier to read for someone who isn't super into types, without much info loss.)~~

edit: Oh, I see. This is a testing specific rollout different from our Trajectory/Transitions